.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,38 @@
-Please include the following information in your ticket.
+Please [read these guidelines](http://ibm.biz/cdt-issue-guide) before opening an issue.
 
-- sync-android version(s) that are affected by this issue.
-- sync-android artifact you are using
+<!-- Issues will be CLOSED IMMEDIATELY if the following template is not completed. -->
+
+## Bug Description
+
+### 1. Steps to reproduce and a small code sample that demonstrates the issue
```

Comment:
How about something like https://gist.github.com/tomblench/909e5ea69428d1da2cb4c8c40b60b5a5 ?

It's a contrived example, and not actually a bug, but at least it's clearer because it is a concrete example from this library